### PR TITLE
Making tests happy.

### PR DESCRIPTION
**Pull request by domesticmouse**
Also trimming whitespace from README.md.

### CODE REVIEW

**Comment by domesticmouse** [comment]
PTAL @markmcd

File: src/test/java/com/google/maps/AuthenticatedTest.java [hunk 1]
```diff
@@ -48,20 +48,30 @@ protected AuthenticatedTest() {
...
+    } else if (System.getProperty("api.key") != null && !System.getProperty("api.key").equals("")) {
       GeoApiContext context = new GeoApiContext()
-          .setEnterpriseCredentials(System.getenv("CLIENT_ID"), System.getenv("CLIENT_SECRET"));
+          .setApiKey(System.getProperty("api.key"));
       contexts.add(new Object[]{context});
     }
 
+    if (supportsClientId) {
+        if (!(System.getenv("CLIENT_ID") == null
+        || System.getenv("CLIENT_SECRET") == null)) {
```
Comment (markmcd) [inline src/test/java/com/google/maps/AuthenticatedTest.java]: nit: continuation spacing here should be +4

**Comment by markmcd** [comment]
LGTM

File: src/test/java/com/google/maps/AuthenticatedTest.java [hunk 2]
```diff
@@ -48,20 +48,30 @@ protected AuthenticatedTest() {
...
+        GeoApiContext context = new GeoApiContext()
+            .setEnterpriseCredentials(System.getenv("CLIENT_ID"), System.getenv("CLIENT_SECRET"));
+        contexts.add(new Object[]{context});
+      } else if (!(System.getProperty("client.id") == null || System.getProperty("client.id").equals("")
+          || System.getProperty("client.secret") == null || System.getProperty("client.secret").equals(""))) {
+          GeoApiContext context = new GeoApiContext()
+              .setEnterpriseCredentials(System.getProperty("client.id"), System.getProperty("client.secret"));
+          contexts.add(new Object[]{context});
```
Comment (broady) [inline src/test/java/com/google/maps/AuthenticatedTest.java]: There must be some way to make this read more nicely. Maybe some local variables?

File: src/test/java/com/google/maps/AuthenticatedTest.java (hunk 1 above)
Comment (domesticmouse) [inline src/test/java/com/google/maps/AuthenticatedTest.java]: Do we have a code style import method for IntelliJ?

File: src/test/java/com/google/maps/AuthenticatedTest.java (hunk 2 above)
Comment (domesticmouse) [inline src/test/java/com/google/maps/AuthenticatedTest.java]: I've introduced local vars. What line length limit are we using for this project?

Comment (broady) [inline src/test/java/com/google/maps/AuthenticatedTest.java]: lg. i think 100-120 is reasonable.

Comment (domesticmouse) [inline src/test/java/com/google/maps/AuthenticatedTest.java]: Thanks Cbro!

**Pull request merged**